app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -1447,6 +1447,8 @@ def build_toolbar_select_button(id)
     return true if id == "drift_all"
     return true if id.starts_with?("comparemode_") && id.ends_with?(@settings[:views][:compare_mode])
     return true if id.starts_with?("driftmode_") && id.ends_with?(@settings[:views][:drift_mode])
+    return true if id == "view_dashboard" && @showtype == "dashboard"
+    return true if id == "view_summary" && @showtype == "main"
```

Comment:
Personally I think this method could benefit of some refactoring. Not for you @zeari and not for now.

@blomquisg @dclarizio I think the cases here could be consolidated, the code improved, etc. it may be good for a list of refactoring you may be maintaining.